pkg/fission-cli/util/util.go

```diff
@@ -372,6 +372,20 @@ func UpdateMapFromStringSlice(dataMap *map[string]string, params []string) bool
 	return updated
 }
 
+// UpdateEnvVarFromStringSlice parses key, val from "key=val" string array and updates passed []v1.EnvVar
+func UpdateEnvVarFromStringSlice(envVar []v1.EnvVar, params []string) bool {
```

Comment:
Should we redefine this function as 

```
func GetEnvVarFromStringS(params []string) (envVar []v1.EnvVar, error)
```

Also, please add unit tests for the function in util_test.go.